api/server/router/image/image_routes.go

```diff
@@ -440,6 +440,8 @@ func (ir *imageRouter) getImagesSearch(ctx context.Context, w http.ResponseWrite
 			return errdefs.InvalidParameter(errors.Wrap(err, "invalid limit specified"))
 		}
 	}
+
+	// TODO(thaJeztah): reset the "is-automated" filter on API v1.45 and up (once released).
```

Comment:
Couldn't we version this change by e.g. making it an invalid request to search using an `is-automated` filter condition on API v1.45+?

> so we should just ignore the filter, and reset the field

We can ignore `is-automated=false` as that will match all images, but we can't just ignore `is-automated=true`. It would be more surprising for a client searching with an `is-automated=true` filter to get results where `"is_automated": false` than to get no results at all.